distributed/client.py

```diff
@@ -2620,6 +2620,11 @@ def get(
             user_priority=priority,
             actors=actors,
         )
+
+        for key in set(flatten([keys])):
```

Comment:
A small point, but instead of doing `set(flatten([keys]))` twice (once here and once in the `_graph_to_futures` call, could you define `keys_flatten = set(flatten([keys]))` first and then use that variable in both places